slatedb/tests/db.rs

```diff
@@ -10,9 +10,19 @@ use std::collections::HashMap;
 use std::sync::atomic::{AtomicU64, Ordering};
 use std::sync::Arc;
 use std::time::Duration;
+use tokio_util::sync::CancellationToken;
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::EnvFilter;
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 8)]
 async fn test_concurrent_writers_and_readers() {
+    let filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("debug"));
```

Comment:
Is this necessary or it’s here for debugging purposes?

Reply:
Ah, I added it so logs are visible with --nocapture. We have a harness that does this for all unit tests, but we have to do it manually here since it's an integration test.